node/src/actors/chain_manager/mod.rs

```diff
@@ -273,14 +280,16 @@ impl ChainManager {
         let chain_beacon = state.get_chain_beacon();
         let superblock_beacon = state.get_superblock_beacon();
 
-        log::debug!(
-            "Persisting chain state for superblock #{} with chain beacon {:?} and super beacon {:?}",
-            superblock_index,
-            chain_beacon,
-            superblock_beacon
-        );
+        if let Some(superblock_index) = superblock_index {
+            log::debug!(
+                "Persisting chain state for superblock #{} with chain beacon {:?} and super beacon {:?}",
+                superblock_index,
+                chain_beacon,
+                superblock_beacon
+            );
 
-        assert_eq!(superblock_beacon.checkpoint, superblock_index);
+            assert_eq!(superblock_beacon.checkpoint, superblock_index);
+        }
```

Comment:
Add a different log when superblock_index is None? "Persisting chain state during synchronization, chain beacon: {:?}"